backend/src/tasks/test/lookingGlass.test.ts

```diff
@@ -0,0 +1,671 @@
+        ref: {
+          type: 'ipv4',
+          id: 86421357
+        },
+        name: '100.123.100.123',
+        type: 'ipv4',
+        threatNames: ['Gumblar Infection'],
+        locations: [
+          {
+            city: 'testland',
+            country: 'USA',
+            region: 'UT',
+            geoPoint: {
+              lon: -55.55,
+              lat: 55.55
+            },
+            countryName: 'United States',
+            sources: ['Source3'],
+            country2Digit: 'US',
+            lastSeen: 1620892865376
+          }
+        ],
+        ipv4: 123459543,
+        asns: [1357],
+        threatIds: ['ThreatID_1234'],
+        cidrv4s: ['123.123.12.0/12', '123.123.0.0/12', '123.0.0.0/1']
+      },
+      ref: {
+        type: 'associated-with',
+        right: {
+          type: 'threat',
+          id: 'ThreatID_1234'
+        },
+        left: {
+          type: 'ipv4',
+          id: 86421357
+        }
+      }
+    },
+    {
+      firstSeen: Today.getTime() - 9 * 24 * 60 * 60 * 1000,
+      lastSeen: Today.getTime() - 7 * 24 * 60 * 60 * 1000,
+      sources: ['Shodan Inferred Vulnerability'],
+      right: {
+        ticClassificationScores: [50],
+        threatId: 'ThreatID_1234',
+        classifications: ['Vulnerable Service'],
+        ticScore: 50,
+        ticCriticality: 50,
+        ticSourceScore: 50,
+        ticObsInfluence: 0.5,
+        ref: {
+          type: 'threat',
+          id: 'ThreatID_1234'
+        },
+        threatCategories: ['Vulnerable Service'],
+        name: 'Vulnerable Product - CiscoWebVPN',
+        type: 'threat',
+        threatName: 'Vulnerable Product - CiscoWebVPN'
+      },
+      left: {
+        collectionIds: [
+          'test_collection_id_1',
+          'test_collection_id_5',
+          '].`test_collection_id_4'
+        ],
+        classifications: ['Vulnerable Service'],
+        ticScore: 50,
+        owners: ['owner1', 'owner2', 'owner3'],
+        ref: {
+          type: 'ipv4',
+          id: 1234509876
+        },
+        name: '1.123.135.123',
+        type: 'ipv4',
+        threatNames: ['Vulnerable Product - CiscoWebVPN'],
+        locations: [
+          {
+            city: 'testland',
+            country: 'USA',
+            region: 'CO',
+            geoPoint: {
+              lon: -55.55,
+              lat: 55.55
+            },
+            countryName: 'United States',
+            sources: ['Source4'],
+            country2Digit: 'US',
+            lastSeen: 1620892865376
+          }
+        ],
+        ipv4: 135798642,
+        asns: [4321],
+        threatIds: ['ThreatID_1234'],
+        cidrv4s: ['123.123.12.0/12', '123.123.0.0/12', '123.0.0.0/1']
+      },
+      ref: {
+        type: 'associated-with',
+        right: {
+          type: 'threat',
+          id: 'ThreatID_1234'
+        },
+        left: {
+          type: 'ipv4',
+          id: 1234509876
+        }
+      }
+    }
+  ]
+};
+let OrgName;
+let org_Response;
```

Comment:
Can you make sure all variables are camel-case here?

org_Response -> orgResponse
Today -> today
OrgName -> orgName

etc.